test/test_entropyrate.py

```diff
@@ -1,48 +1,62 @@
 import unittest
-from pyinform import activeinfo
+from pyinform import entropyrate
 from math import isnan
+import numpy
 
-class TestActiveInfo(unittest.TestCase):
+class EntropyRateHistoryTooLong(unittest.TestCase):
 
-    def testSeriesTooShort(self):
+    def testHistoryTooLongBase2(self):
+        series = numpy.random.randint(2,size = 30)
         with self.assertRaises(ValueError):
-            activeinfo([],2)
+            entropyrate(series,26,2)
 
+    def testHistoryTooLongBase3(self):
+        series = numpy.random.randint(3,size = 30)
         with self.assertRaises(ValueError):
-            activeinfo([1],2)
+            entropyrate(series,16,3)
 
-    def testHistoryLengthTooShort(self):
+    def testHistoryTooLongBase4(self):
+        series = numpy.random.randint(4,size = 30)
         with self.assertRaises(ValueError):
-            activeinfo([0,1,1,0,0,1,0], 0)
+            entropyrate(series,13,4)
 
-    def testEncodingError(self):
-        series = [2,1,0,0,1,0,0,1]
-        activeinfo(series, 2, 3)
+class TimeSeriesTooShort(unittest.TestCase):
```

Comment:
You might want to make this name more specific, e.g. `EntropyRateTimeSeriesTooShort`. The problem is that if you have a `TimeSeriesTooShort` class in a different file testing, say Transfer Entropy, then these names would clash and the tests would probably break.